jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/JDTUtils.java

```diff
@@ -647,17 +649,24 @@ public static boolean isValidLevel1URI(String uriString) {
     /**
      * Returns a list of all accessors (getter and setter) of the given field.
      * 
-     * @param field
+     * @param unit, field
```

Comment:
Normally these Javadoc comments are read by a tool which expects each argument to have its own @param annotation. Even if we don't use this tool, this method is now long enough it makes sense to write a short description of the arguments (one per line).

Reply:
Done